### PR TITLE
Started adding Measurement

### PR DESCRIPTION
**Pull request by ThomasRanvikEriksen**
Started adding Measurement
Need to check some things
Need to move CT and PT
Missing som relations like Quality

Do not merge this yet!

### CODE REVIEW

File: DIGIN10/Grid/CIMXML/DIGIN10-30-LV1_OP.xml [hunk 1]
```diff
@@ -18,26 +18,88 @@
...
+    <cim:CommunicationLink rdf:ID="_1fe5049c-34d7-4e30-a411-ff01c16b8f3a">
+        <cim:IdentifiedObject.mRID>1fe5049c-34d7-4e30-a411-ff01c16b8f3a</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>Disruptive Technolgies WiFi</cim:IdentifiedObject.description>
+        <cim:IdentifiedObject.name>DisrTechWiFi</cim:IdentifiedObject.name>
+        <cim:CommunicationLink.RemoteUnits rdf:resource="#_00160296-775f-407a-8d67-c9a5d610715a"/>
+    </cim:CommunicationLink>
+
+    <cim:CommunicationLink rdf:ID="_2bf1f9fc-263d-4995-80da-9802dc087fe5">
+        <cim:IdentifiedObject.mRID>2bf1f9fc-263d-4995-80da-9802dc087fe5</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>Disruptive Technolgies WiFi</cim:IdentifiedObject.description>
```
Comment (ThomasRanvikEriksen) [inline DIGIN10/Grid/CIMXML/DIGIN10-30-LV1_OP.xml:59]: Wifi should be replaced with 4G

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        <md:Model.scenarioTime>2022-03-30T23:00:00Z</md:Model.scenarioTime>
+        <md:Model.description>DIGIN10 Medium Voltage 1 (MV1) Asset Model</md:Model.description>
+        <md:Model.modelingAuthoritySet>https://diginenergi.no/DIGIN10/Asset</md:Model.modelingAuthoritySet>
+        <md:Model.profile>http://iec.ch/TC57/CIM/Asset/2/0</md:Model.profile>
+        <md:Model.version>1</md:Model.version>
+    </md:FullModel>
+
+    <cim:Asset rdf:ID="_3344a693-6818-4703-93ac-510baac566d5">
+        <cim:IdentifiedObject.mRID>3344a693-6818-4703-93ac-510baac566d5</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Current Transformer 1 Asset 3</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:19]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 CT1 Asset3</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:20]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        <cim:Asset.purchasePrice>100000</cim:Asset.purchasePrice>
+        <cim:Asset.serialNumber>8DuSTPRi</cim:Asset.serialNumber>
+        <cim:Asset.type>CurrentTransformer</cim:Asset.type>
+        <cim:Asset.utcNumber>10000000000003</cim:Asset.utcNumber>
+        <cim:Asset.PowerSystemResources rdf:resource="#_26d2f8aa-b651-46c5-a54d-05948f35f3d2" />
+    </cim:Asset>
+
+    <cim:Asset rdf:ID="_9e9d30f5-fdb3-4d0e-9309-ad41c97f3df5">
+        <cim:IdentifiedObject.mRID>9e9d30f5-fdb3-4d0e-9309-ad41c97f3df5</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Current Transformer 2 Asset 4</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:38]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 CT2 Asset4</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:39]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        <cim:Asset.purchasePrice>100000</cim:Asset.purchasePrice>
+        <cim:Asset.serialNumber>8DuSMPRi</cim:Asset.serialNumber>
+        <cim:Asset.type>CurrentTransformer</cim:Asset.type>
+        <cim:Asset.utcNumber>10000000000004</cim:Asset.utcNumber>
+        <cim:Asset.PowerSystemResources rdf:resource="#_0f3b6200-684d-4365-8d55-45baae73bffb" />
+    </cim:Asset>
+
+    <cim:Asset rdf:ID="_addaff9d-8a88-4f0b-8213-1a15498083a2">
+        <cim:IdentifiedObject.mRID>addaff9d-8a88-4f0b-8213-1a15498083a2</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Current Transformer 3 Asset 5</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:57]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 CT3 Asset5</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:58]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+        <cim:Asset.purchasePrice>100000</cim:Asset.purchasePrice>
+        <cim:Asset.serialNumber>8DuSTNRi</cim:Asset.serialNumber>
+        <cim:Asset.type>CurrentTransformer</cim:Asset.type>
+        <cim:Asset.utcNumber>10000000000005</cim:Asset.utcNumber>
+        <cim:Asset.PowerSystemResources rdf:resource="#_cfb857c9-3ac2-4ea8-bcdb-1e50bd15f541" />
+    </cim:Asset>
+
+    <cim:Asset rdf:ID="_ff20c614-4797-471e-98c6-c08f7ed97ce6">
+        <cim:IdentifiedObject.mRID>ff20c614-4797-471e-98c6-c08f7ed97ce6</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Potential Transformer 1 Asset 6</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:76]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 PoT1 Asset6</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:77]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+        <cim:Asset.purchasePrice>100000</cim:Asset.purchasePrice>
+        <cim:Asset.serialNumber>8HuPTPRy</cim:Asset.serialNumber>
+        <cim:Asset.type>PotentialTransformer</cim:Asset.type>
+        <cim:Asset.utcNumber>10000000000007</cim:Asset.utcNumber>
+        <cim:Asset.PowerSystemResources rdf:resource="#_fd11bf7f-138b-4a1b-802c-25262b7b8672" />
+    </cim:Asset>
+
+    <cim:Asset rdf:ID="_0507d6c7-b845-4171-ad56-d531e2e52401">
+        <cim:IdentifiedObject.mRID>ff20c614-4797-471e-98c6-c08f7ed97ce6</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Potential Transformer 2 Asset 7</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:95]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 PoT2 Asset7</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:96]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+        <cim:Asset.purchasePrice>100000</cim:Asset.purchasePrice>
+        <cim:Asset.serialNumber>8YuPTPRy</cim:Asset.serialNumber>
+        <cim:Asset.type>PotentialTransformer</cim:Asset.type>
+        <cim:Asset.utcNumber>10000000000008</cim:Asset.utcNumber>
+        <cim:Asset.PowerSystemResources rdf:resource="#_03f7a245-aeca-48ba-af07-58419fb9b77c" />
+    </cim:Asset>
+
+    <cim:Asset rdf:ID="_1104d9f6-95d3-463a-b8ad-6f64eb3a730b">
+        <cim:IdentifiedObject.mRID>ff20c614-4797-471e-98c6-c08f7ed97ce6</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>11 kilo Volt Vestfoldstien 2 Potential Transformer 3 Asset 8</cim:IdentifiedObject.description>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:114]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+		<cim:IdentifiedObject.name>11 VESTFO 2 PoT3 Asset8</cim:IdentifiedObject.name>
```
Comment (geiabu) [inline DIGIN10/Asset/DIGIN10-30-MV1-Asset.xml:115]: Change from 11 kilo Volt to 22 kV

File: DIGIN10/Grid/CIMXML/DIGIN10-30-MV1_OP.xml [hunk 14]
```diff
@@ -17,27 +17,256 @@
...
+        <cim:Measurement.measurementType>PhaseCurrent</cim:Measurement.measurementType>
+        <cim:Measurement.phases rdf:recource="http://iec.ch/TC57/CIM100#PhaseCode.C" />
+        <cim:Measurement.unitMultiplier rdf:recource="http://iec.ch/TC57/CIM100#UnitMultiplier.none" />
+        <cim:Measurement.unitSymbol rdf:recource="http://iec.ch/TC57/CIM100#UnitSymbol.A" />
+        <cim:Analog.maxValue>450</cim:Analog.maxValue>
+        <cim:Analog.minValue>0</cim:Analog.minValue>
+        <cim:Analog.normalValue>270</cim:Analog.normalValue>
+        <cim:Analog.positiveFlowIn>false</cim:Analog.positiveFlowIn>
+        <cim:Measurement.Asset rdf:resource="#_3344a693-6818-4703-93ac-510baac566d5" />
+        <cim:Measurement.PowerSystemResources rdf:resource="#_cfb857c9-3ac2-4ea8-bcdb-1e50bd15f541" /> <!--Need to check this-->
```
Comment (Joahimmehl) [inline DIGIN10/Grid/CIMXML/DIGIN10-30-MV1_OP.xml:68]: Checked?

**Review by geiabu — APPROVED** [review]

**Review by geiabu — APPROVED** [review]

**Pull request merged**